target/bin/open-dkim

```diff
@@ -144,6 +144,9 @@ while read -r DKIM_DOMAIN; do
       --directory="/tmp/docker-mailserver/opendkim/keys/${DKIM_DOMAIN}"
   fi
 
+  # fix permissions to use the same use as /tmp/docker-mailserver/opendkim/keys
```

Comment:
```suggestion
  # fix permissions to use the same as /tmp/docker-mailserver/opendkim/keys
```

Reply:
fixed with:
```
# fix permissions to use the same user:group as /tmp/docker-mailserver/opendkim/keys
```
somehow during testing i didn't fully finish this comment